org.adempiere.base/src/org/compiere/model/MRefList.java

```diff
@@ -303,7 +303,8 @@ else if ("V".equals(orderBy))
 			.append("FROM AD_Ref_List INNER JOIN AD_Ref_List_Trl trl ")
 			.append(" ON (AD_Ref_List.AD_Ref_List_ID=trl.AD_Ref_List_ID AND trl.AD_Language='")
 			.append(language).append("')");
-		sql.append(" WHERE AD_Ref_List.AD_Reference_ID=").append(AD_Reference_ID);
+		sql.append(" WHERE AD_Ref_List.AD_Reference_ID=").append(AD_Reference_ID)

```

Comment:
As noted by @nmicoud 
The change here must be consistent with https://wiki.idempiere.org/en/NF11_Enable_Showing_Inactive_Records_through_Reference

Reply:
Ok